### DISCUSSION TITLE
hi developers, will there be a pantheon edtion for SpriralLinux?

### DISCUSSION THREAD

**Discussion by huijingfei** [SpiralLinux/SpiralLinux-project#31] [Q&A] [answered]
I notice that there is a pantheon edition for GeckoLinux, it's perfect to have a pantheon desktop on Debian, but there isn't yet. I'm a normal user so I don't have the ability to install pantheon on official Debian, wish you can add it.
I ever used elementary os for a long time, buy they changed the packages from deb to flatpak since release 6, I don't like it. 
Open source is great, thanks to all the developers.

**Comment by geckolinux** [comment, marked as answer]
Hi @huijingfei , thanks for the feedback. Unfortunately that will not be possible until Pantheon packages exist in the Debian repository.

**Reply by huijingfei** [reply]
thanks for your answer, I search the difference between debian and opensuse on google, I think opensuse will be another perfect distro for me.

**Reply by moebeer** [reply]
I'm also a Pantheon fan but whenever I use it I miss the Global menu so I installed SpiralLinux with KDE and have pretty much duplicated the Pantheon interface.

**Reply by geckolinux** [reply]
@moebeer Very cool, I'd love to see some screenshots.

**Reply by moebeer** [reply]
Sure thing!
![imitating-eos2](https://user-images.githubusercontent.com/108588090/177891568-07228a91-adc3-47ca-8dd6-6fe00818ea71.png)
![imitating-eos1](https://user-images.githubusercontent.com/108588090/177891578-7c1bcbe0-3b35-4412-aae9-0cc9e824e4f3.png)
![imitating-eos3](https://user-images.githubusercontent.com/108588090/177891589-e1fc8039-1869-4451-aaa2-8e44599a6ca6.png)
![imitating-eos4](https://user-images.githubusercontent.com/108588090/177891599-af5ec876-00af-4d39-b0e6-e03264ad2d05.png)

**Reply by geckolinux** [reply]
@moebeer  Wow, well done. Thanks!
